roles/setup/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+# tasks file for Setup
+
+  - name: Install pip from apt
+    apt: 
+      name: python-pip
+      state: present
+
+  - name: Update APT package cache
+    apt: 
+      update_cache: yes
```

Comment:
edw px tha mporouses na to enwseis me to apo panw...kai genika, prwta tha theleis na kaneis update to package cache kai meta na kaneis install pragmata gia na pareis tin latest version